tools/README.md

```diff
@@ -167,3 +167,64 @@ The output chunked graph metadata will go as follows (assuming the current direc
     "edge_data": {}
 }
 ```
+
+## Partition Config Conversion
+
+`convert_partition_conf.py` is a tool help you convert old partition config to new version.
+
+### Sample usage like
+
+```
+python tools/convert_partition_conf.py --part-config "{config path}"
+```
+
+### Requirement
+
+The machine run this script require **part 0 data is accessible under the same folder as part_config**.
+
+### Potential risk
+
+The tool may fail in some extrema unbalanced situation where partition 0 doesn't contain any edges of each edge type and corresponding partition data is in-accessible from 
+current place, which may happen rarely.
```

Comment:
I won't put this in the doc. Instead, your script should throw proper error messages for the users.